api/envoy/extensions/filters/http/ext_proc/v3/ext_proc.proto

```diff
@@ -180,6 +181,17 @@ message ExternalProcessor {
   // :ref:`clear_route_cache <envoy_v3_api_field_service.ext_proc.v3.CommonResponse.clear_route_cache>`
   // field is set in an external processor response.
   bool disable_clear_route_cache = 11;
+
+  // [#not-implemented-hide:]
+  // Allow headers matching the ``forward_rules`` to be forwarded to the external processing server.
+  // If not set, all headers are forwarded to the external processing server.
+  HeaderForwardingRules forward_rules = 12;
+}
+
+// The HeaderForwardingRules structure specifies what headers are
+// allowed to be forwarded to the external processing server.
+message HeaderForwardingRules {
+  type.matcher.v3.ListStringMatcher allowed_headers = 1;
```

Comment:
nit: please add a short description for the field.

Reply:
I tried to add a few words for the field, but it is pretty much duplicate with what's before the message.

Reply:
Please recommend.

Reply:
We can modify the text in the future if there are more fields added into HeaderForwardingRules. How about let's stay as it is for now?

Reply:
Ok.